test/provision/k8s_install.sh

```diff
@@ -310,6 +316,11 @@ if [[ "${HOST}" == "k8s1" ]]; then
       sudo cp -i /etc/kubernetes/admin.conf /root/.kube/config
       sudo chown root:root /root/.kube/config
 
+      if [[ "${KUBEPROXY}" == "0" ]]; then
+          kubectl -n kube-system delete ds kube-proxy
+          iptables-restore <(iptables-save | grep -v KUBE)
+      fi
```

Comment:
This means we have only deleted the iptables rules in k8s1, not on all nodes.

Reply:
It's not provisioned on other nodes.